src/web/app/pages-admin/admin-home-page/admin-home-page.component.ts

```diff
@@ -64,12 +58,18 @@ export class AdminHomePageComponent implements OnInit {
         invalidLines.push(instructorDetail);
         continue;
       }
-      this.instructorsConsolidated.push({
-        name: instructorDetailSplit[0],
-        email: instructorDetailSplit[1],
-        institution: instructorDetailSplit[2],
-        status: 'PENDING',
-        isCurrentlyBeingEdited: false,
+
+      const requestData: AccountCreateRequest = {
+        instructorEmail: instructorDetailSplit[1],
+        instructorName: instructorDetailSplit[0],
+        instructorInstitution: instructorDetailSplit[2],
+      };
+
+      this.accountService.createAccountRequest(requestData)
+      .subscribe({
+        next: () => {
+          this.fetchAccountRequests();
```

Comment:
This makes an API call for each account request, I think you can just call this once at the end of the entire loop after all requests are processed, since this is a batch creation feature anyway

Reply:
I ran into an issue where the account requests are not updated if the fetch is moved outside of the subscribe, I'm assuming because the API call is not instant - logging seemed to show that was happening.
What is the recommended way of waiting for the requests to finish before moving onto the next statement?

Reply:
- Moved `createAccountRequest()` into a promise and push to an array of promises
- Moved `fetchAccountRequests()` into an allSettled thenable.